### PR TITLE
KAZOO-4369: Add account_id to app_store API return & update kz_apps_s…

### PR DESCRIPTION
**Pull request by macpie**
…tore to kzd

### CODE REVIEW

File: core/kazoo_documents-1.0.0/src/kzd_app.erl [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+%%% @contributors
+%%%   Peter Defebvre
+%%%-------------------------------------------------------------------
+-module(kzd_app).
+
+-export([fetch/2]).
+
+-export([
+    id/1
+    ,published/1
```
Comment (fenollp) [inline core/kazoo_documents-1.0.0/src/kzd_app.erl]: How about `is_published`?

File: core/kazoo_documents-1.0.0/src/kzd_app.erl [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+-spec id(wh_json:object()) -> ne_binary().
+id(JObj) ->
+    wh_doc:id(JObj).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec published(wh_json:object()) -> ne_binary().
```
Comment (fenollp) [inline core/kazoo_documents-1.0.0/src/kzd_app.erl]: Wrong return type

File: core/kazoo_documents-1.0.0/src/kzd_app.erl [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+    wh_json:is_true(?PUBLISHED, JObj, 'true').
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec name(wh_json:object()) -> ne_binary().
+name(JObj) ->
+    wh_json:get_binary_value(?NAME, JObj).
```
Comment (fenollp) [inline core/kazoo_documents-1.0.0/src/kzd_app.erl]: This can be `<<>>`

File: core/kazoo_documents-1.0.0/src/kzd_app.erl [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+-spec screenshots(wh_json:object()) -> ne_binary().
+screenshots(JObj) ->
+    wh_json:get_list_value(?SCREENSHOTS, JObj).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec urls(wh_json:object()) -> ne_binary().
```
Comment (fenollp) [inline core/kazoo_documents-1.0.0/src/kzd_app.erl]: All above return types

File: core/kazoo_documents-1.0.0/src/kzd_app.erl (hunk 3 above)
Comment (macpie) [inline core/kazoo_documents-1.0.0/src/kzd_app.erl]: I ll change it to get_ne_binary

File: core/kazoo_documents-1.0.0/src/kzd_app.erl (hunk 1 above)
Comment (macpie) [inline core/kazoo_documents-1.0.0/src/kzd_app.erl]: sounds good

File: core/kazoo_documents-1.0.0/src/kzd_app.erl (hunk 4 above)
Comment (macpie) [inline core/kazoo_documents-1.0.0/src/kzd_app.erl]: Fixed

File: core/kazoo_documents-1.0.0/src/kzd_app.erl [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+-spec license(wh_json:object()) -> ne_binary().
+license(JObj) ->
+    wh_json:get_ne_binary_value(?LICENSE, JObj).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec price(wh_json:object()) -> ne_binary().
```
Comment (fenollp) [inline core/kazoo_documents-1.0.0/src/kzd_app.erl]: You missed some

File: core/kazoo_documents-1.0.0/src/kzd_app.erl [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+-spec id(wh_json:object()) -> ne_binary().
+id(JObj) ->
+    wh_doc:id(JObj).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec is_published(wh_json:object()) -> ne_binary().
```
Comment (jamesaimonetti) [inline core/kazoo_documents-1.0.0/src/kzd_app.erl]: Wrong return type

File: core/kazoo_documents-1.0.0/src/kzd_app.erl [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+-spec name(wh_json:object()) -> ne_binary().
+name(JObj) ->
+    wh_json:get_ne_binary_value(?NAME, JObj).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec i18n(wh_json:object()) -> ne_binary().
```
Comment (jamesaimonetti) [inline core/kazoo_documents-1.0.0/src/kzd_app.erl]: Wrong return type

File: core/kazoo_documents-1.0.0/src/kzd_app.erl [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+i18n(JObj) ->
+    wh_json:get_json_value(?I18N, JObj).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec tags(wh_json:object()) -> ne_binary().
```
Comment (jamesaimonetti) [inline core/kazoo_documents-1.0.0/src/kzd_app.erl]: Wrong return type

File: core/kazoo_documents-1.0.0/src/kzd_app.erl [hunk 9]
```diff
@@ -0,0 +1,195 @@
...
+-spec price(wh_json:object()) -> api_float().
+price(JObj) ->
+    wh_json:get_float_value(?PRICE, JObj).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec screenshots(wh_json:object()) -> ne_binary().
```
Comment (jamesaimonetti) [inline core/kazoo_documents-1.0.0/src/kzd_app.erl]: Wrong return type

File: core/kazoo_documents-1.0.0/src/kzd_app.erl (hunk 4 above)
Comment (jamesaimonetti) [inline core/kazoo_documents-1.0.0/src/kzd_app.erl]: This one isn't fixed.

Comment (jamesaimonetti) [inline core/kazoo_documents-1.0.0/src/kzd_app.erl]: Bad copy/paste, bad!

File: core/kazoo_documents-1.0.0/src/kzd_app.erl (hunk 7 above)
Comment (jamesaimonetti) [inline core/kazoo_documents-1.0.0/src/kzd_app.erl]: Still wrong.

Comment (jamesaimonetti) [inline core/kazoo_documents-1.0.0/src/kzd_app.erl]: Can you also fix the spec in kzd_apps_store: "kzd_apps_store.erl:97: Invalid type specification for function kzd_apps_store:set_blacklist/2"

**Pull request merged**